scripts/gee

```diff
@@ -993,15 +993,17 @@ function _install_tools() {
       | sudo /usr/bin/apt-key --keyring "${KRFILE}" add -
     echo "deb [arch=amd64 signed-by=${KRFILE}] https://cli.github.com/packages stable main" \
       | sudo /usr/bin/tee /etc/apt/sources.list.d/githubcli-archive.list
-    sudo /usr/bin/apt update && sudo /usr/bin/apt install gh
+    sudo /usr/bin/apt update || /bin/true  # ignore errors associated with stale repos.
```

Comment:
also minor nit: I've seen `apt-get` still being preferred in shell script.

This [not very loved stackoverflow answer](https://unix.stackexchange.com/questions/590699/should-i-use-apt-or-apt-get-in-shell-scripting) provides the rationale so I don't have to type it (source: apt man page).

Reply:
Done.